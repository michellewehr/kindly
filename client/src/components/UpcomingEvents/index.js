


export default function UpcomingEvents() {
  return (
    <div>
      <div class="bg-sky-100 mx-auto text-center rounded w-9/12 mb-2">
        <h2 class="mb-1 underline">Upcoming Registered Events</h2>
        {/* div for each event */}
        <a class="bg-sky-100" href="">
          <div class="text-left px-1">
            <h3 class="bg-cyan-600 hover:bg-orange-300">Event Name</h3>
            <p>Location</p>
            <p>Date at Start Time- End Time</p>
          </div>
        </a>
        {/* end of div for each event */}
        
      </div>
    </div>
  );
}

