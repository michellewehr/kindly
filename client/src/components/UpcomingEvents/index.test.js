import { render, cleanup, fireEvent } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import UpcomingEvents from './index';

afterEach(cleanup);

describe('UpcomingEvents component renders', () => {

   it('renders', () => {
      render(<UpcomingEvents />);
   });

   it('Matches snapshot DOM node structure', () => {
      const { asFragment } = render(<UpcomingEvents />);
      expect(asFragment()).toMatchSnapshot();
   });
});