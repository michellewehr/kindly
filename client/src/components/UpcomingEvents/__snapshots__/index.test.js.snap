// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UpcomingEvents component renders Matches snapshot DOM node structure 1`] = `
<DocumentFragment>
  <div>
    <div
      className="bg-sky-100 mx-auto text-center rounded w-9/12 mb-2"
    >
      <h2
        className="mb-1 underline"
      >
        Upcoming Registered Events
      </h2>
      <a
        className="bg-sky-100"
        href=""
      >
        <div
          className="text-left px-1"
        >
          //! get upcoming event data
          <h3
            className="bg-cyan-600 hover:bg-orange-300"
          >
            Event Name
          </h3>
          <p>
            Location
          </p>
          <p>
            Date at Start Time- End Time
          </p>
        </div>
      </a>
    </div>
  </div>
</DocumentFragment>
`;
